### PR TITLE
fixed the partnership page for both laptop and phone size

### PR DESCRIPTION
**Pull request by josephchimebuka**
I made the partnership page responsive on all screen sizes

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved robustness of the metadata configuration by ensuring it only constructs a URL when the relevant environment variable is present.

- **Style Improvements**
	- Enhanced layout and alignment in the stats component for better visual presentation.
	- Modified the partnership styles for improved responsiveness and alignment of elements.

- **Content Updates**
	- Updated the title in the partnership page component to reflect a singular focus on a quest.
	- Introduced a new environment variable to enhance configuration options for the application.

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const metadata: Metadata = {
   title: "Starknet Quest",
   description:
     "Starknet Quest helps protocols attract and retain users by creating gamified quest experiences on Starknet.",
-  metadataBase: new URL(process.env.NEXT_PUBLIC_APP_LINK as string),
+  // metadataBase: new URL(process.env.NEXT_PUBLIC_APP_LINK as string),
```
Comment (fricoben) [inline app/layout.tsx]: please don't comment this (you can add `NEXT_PUBLIC_APP_LINK =https://starknet.quest/` to make it work

File: styles/components/stats.module.css [hunk 2]
```diff
@@ -14,10 +14,13 @@
...
   padding: 40px 0px;
   flex-direction: column;
-  align-items: flex-start;
+  align-items: center;
   gap: 32px;
   max-width: 950px;
   width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
```
[the remark this answers is not in the document]
Comment (fricoben) [reply styles/components/stats.module.css]: Agree, please fix

**Review by fricoben — CHANGES_REQUESTED** [review]
Some comments

**Comment by josephchimebuka** [comment]
@fricoben I have added the metadataBase

File: .env.example [hunk 3]
```diff
@@ -18,4 +18,4 @@ TWITTER_TOKEN=YOUR_TWITTER_TOKEN_HERE
 MONGODB_DBNAME=YOUR_DB_NAME_HERE
 MONGODB_LINK=YOUR_CONNECTION_STRING_HERE
 NEXT_PUBLIC_STARKNET_ID_API_LINK=https://sepolia.api.starknet.id
-NEXT_PUBLIC_STARKSCAN=YOUR_KEY_HERE
\ No newline at end of file
+NEXT_PUBLIC_STARKSCAN=YOUR_KEY_HERE
```
Comment (fricoben) [inline .env.example:21]: Please add `NEXT_PUBLIC_APP_LINK =https://starknet.quest/`

File: next.config.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ module.exports = module.exports = withAxiom({
   },
   images: {
     domains: [
-      "www.starknet.id",
+      "www.starknet.id",
```
Comment (fricoben) [inline next.config.js:14]: delete space

File: styles/components/stats.module.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (fricoben) [reply styles/components/stats.module.css]: @josephchimebuka please fix again

File: app/layout.tsx [hunk 5]
```diff
@@ -19,7 +19,7 @@ export const metadata: Metadata = {
     images: [
       {
         url: "/visuals/starknetquest.webp",
-        width: 680,
+        width: 680,
```
Comment (fricoben) [inline app/layout.tsx:22]: delete space

**Review by fricoben — CHANGES_REQUESTED** [review]
Some code changes

**Comment by josephchimebuka** [comment]
added new changes as requested @fricoben

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**